ui/components/ui/custom-spending-cap/index.scss

```diff
@@ -0,0 +1,25 @@
+.custom-spending-cap {
+  width: 348px;
+
+  &__input-value-and-token-name {
+    display: contents;
+  }
+
+  &__input {
+    width: 280px;
```

Comment:
@georgewrmarshall Yes, I wrapped div in storybook and set the width there to display how it might be in the app. I removed that width's and the parent container is now responsible. Done [here](https://github.com/MetaMask/metamask-extension/pull/15522/commits/190f93d7a3b816927c6d40f7039080fdc87be55b).